Add customReducers to TestContext (#6067)
marmelab/react-admin: packages/ra-test/README.md

});
```

### Using the 'renderWithRedux' wrapper function

> Instead of wrapping the component under test with the `TestContext` by yourself you can get use of all of the above options and test your components almost natively looking like using just `@testing-library/react` thanks to the `renderWithRedux` wrapper function.

Reviewer: ```suggestion
Instead of wrapping the component under test with the `TestContext` by yourself you can use of all of the above options and test your components almost like using just `@testing-library/react` thanks to the `renderWithRedux` wrapper function.
```

Author: I didn't find a way to edit your suggestion so I commited the changes as a separate commit. I agree with the suggested edits and believe the 'of' after 'by yourself you can use' part is not needed.

> // to the way using

Reviewer: I don't get this line

Author: My point was to explain that using renderWithRedux is actually doing the same stuff as wrapping your component in TestContext but more convinient. Line 106 is pushed by mistake, it should have been deleted.

Author: I have added one more commit where it is deleted

> It will return the same output as `@testing-library/react`'s  `render` method with added `dispatch` and `reduxStore` helpers

Reviewer: ```suggestion
It will return the same output as the `render` method from `@testing-library/react` but will add the `dispatch` and `reduxStore` helpers.
```
